libraries/ms-common/inc/spi.h

```diff
@@ -0,0 +1,80 @@
+// HAL SPI
```

Comment:
It doesn't need to be super fancy. I find usage examples or a brief explanation of what the module's supposed to do can be helpful.